### PR TITLE
Fixes #28445: Update require-dir dependency

### PR DESCRIPTION
**Pull request by ehelms**
This is designed to fix the angular-ui tests. But currently the react-ui tests are also failing.

### CODE REVIEW

**Comment by ehelms** [comment]
Now also includes a fix for the broken UI snapshots in the React UI as well. This is designed to fix all current CI issues.

**Comment by ekohl** [comment]
I think the grunt failure might be due to us installing a very old version on our slaves that's no longer compatible with node 12.

File: engines/bastion_katello/package.json [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+    "grunt-htmlhint": "~0.4.1",
+    "grunt-karma": "^1.0.0",
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.22",
+    "karma-coverage": "^1.0.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-ng-html2js-preprocessor": "^1.0.0",
+    "karma-phantomjs-launcher": "^1.0.0",
+    "require-dir": "~1.2.0",
+    "phantomjs-prebuilt": "2.1.10"
```
Comment (ehelms) [inline engines/bastion_katello/package.json]: Re-defining all of these here is kinda ugly, but this code rarely changes so I think it's OK. Something in updating to NodeJS 10+ seems to have affected how all the relative imports work with Grunt.

File: webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap [hunk 2]
```diff
@@ -7,7 +7,7 @@ exports[`Content Details Info should render and contain appropriate components 1
     loadingText="Loading"
     timeout={300}
   >
-    <ForwardRef
+    <Uncontrolled(TabContainer)
```
Comment (waldenraines) [inline webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap]: Is this intentional?  Seems Odd.

Comment (jeremylenz) [reply webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap]: @waldenraines I think this is related to a patternfly change.

Comment (ehelms) [reply webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap]: Foreman had similar changes once vendor v3 was released -- https://github.com/theforeman/foreman/commit/ae55b7eff2f2e653981781d3b73fc7d1361f1952

**Comment by jturel** [comment]
Any interest in assuaging prprocesser?

**Comment by ehelms** [comment]
Fixed prprocessor

**Pull request merged**